extensions/extensions-helm/aissemble-spark-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,130 @@
+{{- if or .Values.rbac.create .Values.rbac.createClusterRole }}
```

Comment:
A: Can we include a template comment on the permissions that are being changed from the default Spark Operator RBAC and why they are necessary? Will help prevent accidental deletion in the future.

Reply:
Let's make it an actual [template comment](https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments) so it doesn't get included in the final resource yaml. Also I don't think the comment is quite correct.

The community Spark Operator chart works perfectly fine without these permissions. Looking back at our offline conversation, I think this is so that the _pipeline invocation service_ can create _instances_ of the SparkApplication CRD (which is novel behavior by the invocation service) to submit pipelines _to the Spark Operator_ for execution.

Reply:
Although, this does make me wonder why we have to update the spark operator service account to affect the operation of the pipeline invocation service.  Seems like probably the long term solution here is that pipeline invocation service should have it's own service account with the minimal permissions it needs instead of sharing with Spark Operator.

Reply:
> Let's make it an actual [template comment](https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments) so it doesn't get included in the final resource yaml. Also I don't think the comment is quite correct.
> 
> The community Spark Operator chart works perfectly fine without these permissions. Looking back at our offline conversation, I think this is so that the _pipeline invocation service_ can create _instances_ of the SparkApplication CRD (which is novel behavior by the invocation service) to submit pipelines _to the Spark Operator_ for execution.

I will change the format to the template comment - I was unfamiliar with this comment type thank you! Also, you are correct, the change is required for the pipeline-invocation-service. I wasn't sure if I should add that information. Making the necessary changes now, thank you Emily.